Ports/julius/package.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port="julius"
+version="1.7.0"
+useconfigure="true"
+files="https://github.com/bvschaik/julius/archive/refs/tags/v${version}.tar.gz v${version}.tar.gz 3ee62699bcbf6c74fe5a9c940c62187141422a9bd98e01747a554fd77483431f"
```

Comment:
The syntax of `files` changed very recently (#20413). Your `files` declaration should now look like this:

```sh
files=(
    "https://github.com/bvschaik/julius/archive/refs/tags/v${version}.tar.gz 3ee62699bcbf6c74fe5a9c940c62187141422a9bd98e01747a554fd77483431f"
)
```